jib-core/src/main/java/com/google/cloud/tools/jib/frontend/CredentialRetrieverFactory.java

```diff
@@ -171,17 +171,17 @@ CredentialRetriever inferCredentialHelper(
                       + inferredCredentialHelperSuffix),
               dockerCredentialHelperFactory);
 
-        } catch (DockerCredentialHelperNotFoundException ex) {
+        } catch (DockerCredentialHelperNotFoundException
+            | NonexistentServerUrlDockerCredentialHelperException ex) {
```

Comment:
How does this failure now affect the build? Just it just continue to some other auth mechanism?

Reply:
Yeah, currently if the first cred helper exists, but doesn't have the correct server url, it'll just fail without checking other cred helpers on the system. So now it will treat cred helpers that are missing the correct url the same way we've been treating missing cred helpers.